sympy/concrete/summations.py

```diff
@@ -96,63 +96,149 @@ class Sum(AddWithLimits, ExprWithIntLimits):
     Examples
     ========
 
+    >>> from sympy import *
+    >>> import math
```

Comment:
Do not use `from sympy import *` or `import math` in SymPy docs. `math` should never be mixed with SymPy and showing it is bad practice. 

Additionally, the SymPy docs do not use `import sympy as sp`, but rather explicitly import individual functions, like `from sympy import Sum, ...`. They should also define symbols using either `sympy.abc` or `symbols`. See https://docs.sympy.org/dev/contributing/new-contributors-guide/writing-tests.html#doctests for more details.

Reply:
@asmeurer  I changed the code. It only uses sympy and only explicitly import individual functions. It passes doctests too.